test/load/README.md

```diff
@@ -0,0 +1,26 @@
+# Load Testing
+
+Load tests aim to test the performance of the system under heavy load. Game server allocation is an example where multiple parallel operations should be tested.
+
+## Build and run test
+
+Prerequisites:
+- Docker.
+- A running k8s cluster.
+- Have kubeconfig file ready.
+
+Load tests are written using Locust. These tests are also integrated with Graphite and Grafana
+for storage and visualization of the results. 
+
+To run load tests using Locust on your local machine:
+
+```
+docker build -t locust-files .
+docker run --rm --network="host" -e "TARGET_HOST=http://127.0.0.1:8001" locust-files:latest
```

Comment:
Out of curiosity are you running on linux ? Last time I checked network host on MacOS wasn't working.

Reply:
Good catch. Yes, I was running on Linux.

I'm thinking to add to the documentation that for macOS and Windows we can use the special DNS name host.docker.internal. Does that sound good?

Reply:
I added this to the README. Please let me know if it is sufficient.